packages/examples/packages/interactive-ui/src/index.ts

```diff
@@ -0,0 +1,201 @@
+import { rpcErrors } from '@metamask/rpc-errors';
+import type {
+  OnHomePageHandler,
+  OnTransactionHandler,
+  OnUserInputHandler,
+} from '@metamask/snaps-sdk';
+import {
+  panel,
+  type OnRpcRequestHandler,
+  heading,
+  button,
+  UserInputEventType,
+  form,
+  input,
+  ButtonType,
+  text,
+  copyable,
+} from '@metamask/snaps-sdk';
+
+/**
+ * Initiate a new interface with the starting screen and return the Snap interface ID.
+ *
+ * @returns The Snap interface ID.
+ */
+export const createInterface = async (): Promise<string> =>
```

Comment:
We usually use regular functions instead of arrow functions for these things.

Then you can use `return await`.